javascript/new-src/bower.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "OLA",
+  "version": "0.9.4",
+  "authors": [
+    "Dave Olsthoorn <dave.olsthoorn@gmail.com>"
+  ],
+  "description": "the OLA web-ui",
+  "keywords": [
+    "dmx",
+    "ola",
+    "open",
+    "lighting",
+    "architecture"
+  ],
+  "license": "MIT",
```

Comment:
I'd prefer this was under the GPL. It prevents companies taking our work and extending it without contributing back.